src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java

```diff
@@ -702,23 +706,76 @@ public void testSubstationFilter() throws Exception {
     }
 
     @Test
-    public void testDuplicateFilter() throws Exception {
+    public void testLineFilterCrud() throws Exception {
         UUID filterId1 = UUID.fromString("99999999-e0c4-413a-8e3e-78e9027d300f");
-        Date modificationDate = new Date();
         LineFilter lineFilter = LineFilter.builder().equipmentID("equipmentID").equipmentName("equipmentName")
             .substationName1("substationName1")
             .substationName2("substationName2").countries1(COUNTRIES1).countries2(COUNTRIES2)
             .nominalVoltage1(new NumericalFilter(RangeType.RANGE, 5., 8.))
             .nominalVoltage2(new NumericalFilter(RangeType.EQUALITY, 6., null))
             .build();
         CriteriaFilter lineCriteriaFilter = new CriteriaFilter(
-                filterId1,
-                modificationDate,
+                null,
+                new Date(),
                 lineFilter
         );
+
+        // --- insert filter --- //
         insertFilter(filterId1, lineCriteriaFilter);
-        mvc.perform(post("/" + FilterApi.API_VERSION + "/filters?duplicateFrom=" + filterId1)).andExpect(status().isOk());
+
+        // check the inserted filter
+        lineCriteriaFilter.setId(filterId1);
```

Comment:
The return value of POST is not a confident expected value to compare with the filter retrieved by a GET, the correct way to compare is using personalized Assertions J as we did with ExpertFilter, cf: [Assertions](https://github.com/gridsuite/filter-server/pull/111/files#diff-4dbf7c4eab219487d699dc1616695aac463376627993460f0ce9090512ede7e9R18)

In others tests, the id is set for the input dto before inserting is not really correct because the id is set in the controller, so not necessary to set id for the input dto